index.template.html

```diff
@@ -3916,6 +3925,191 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
 
   </section>
 
+  <section id="thing-templates" class="normative">
+  <h1>Thing Model</h1>
+      
+  <p class="note">
+    The following section has its origin in [[wot-thing-description]], Annex C. Here Thing Description 
+    Template is renamed to <a>Thing Model</a>, but keeps the same intention.
+    For this version of the specification, <a>Thing Model</a> and its model features (e.g., extendions, obligations, placeholder) are formal 
+    introduced.
+  </p>
+
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
+    driven in <a href="#sec-vocabulary-definition"></a> and <a href="#sec-td-serialization"></a>. Thing Model samples can 
+    also be seen as a subset of the <a>Thing Descriptions</a> information model, which may not contain, 
+    for example, concrete protocol and security information.
```

Comment:
Probably we want to say "It may also contain URL templates rather than concrete URLs" although I guess we have to discuss this.

Reply:
we can add this information after this sentence

Reply:
done